ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-rhoam-addons__v4.13.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: co-exist-rosa-rhods-rhoam
+  cron: '@yearly'
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
+      ADDON2_CONFIG: name=managed-api-service;addon-managed-api-service=1;rosa-cli-required=true;notification-email=interop-qe-ms@redhat.com;rosa=true
+      OPENSHIFT_VERSION: "4.13"
+      PARALLEL: ""
```

Comment:
please use `"false"` (updated the logic to be more strict in https://github.com/openshift/release/pull/40961)

Reply:
Thanks